core/src/main/java/org/infinispan/commands/VisitableCommand.java

```diff
@@ -48,15 +48,23 @@
    boolean ignoreCommandOnStatus(ComponentStatus status);
 
    /**
-    * @return {@code true} if the command needs to read the previous values of the keys it acts on.
+    * @return Nodes on which the command needs to read the previous values of the keys it acts on,
+    * or <code>null</code> if an invocation does not make sense.
```

Comment:
I don't see anything outdated - `loadType()` returns `PRIMARY`, `OWNERS`, or `DONT_LOAD` (that means empty set of nodes).